internal/app/handlers/handlers.go

```diff
@@ -0,0 +1,66 @@
+package handlers
+
+import (
+	"io"
+
+	"net/http"
+
+	"github.com/kripsy/shortener/internal/app/utils"
+)
+
+type Repository interface {
+	CreateOrGetFromMemory(url []byte) (string, error)
+	GetFromMemory(url []byte) (string, error)
+}
+
+// SaveURLHandler — save original url, create short url into memory
+func SaveURLHandler(myMemory Repository, globalURL string) http.HandlerFunc {
```

Comment:
It would be better if you simplify these functions without Go Anonymous Functions.
You can move it to another package or use a struct.

Reply:
corrected, thank u!)